io/EptReader.cpp

```diff
@@ -767,5 +769,82 @@ Dimension::Type EptReader::getCoercedTypeTest(const NL::json& j)
     return getCoercedType(j);
 }
 
+//============================================================================================
+// Pravin: APIs from here are written to make EPT Reader streamable.
+
+void EptReader::loadNextOverlap()
+{
+    std::map<Key, uint64_t>::iterator itr = m_overlaps.begin();
+    std::advance(itr, m_nodeId-1);
+    Key key(itr->first);
+    log()->get(LogLevel::Debug)
+        << "Streaming Data " << m_nodeId << "/" << m_overlaps.size() << ": "
+        << key.toString() << std::endl;
+
+    uint64_t startId(0);
+	m_pointTable.reset(new PointTable());
```

Comment:
Better to write:
```
m_pointTable.reset(new PointTable(m_pointTable->layout()))
```
Then you don't need the for loop below

Reply:
`PointTable` donot have constructor to accept `PointLayoutPtr`.